code/modules/spells/aoe_turf/conjure/arcane_golem.dm

```diff
@@ -72,6 +64,8 @@
 
 	if(!istype(spell_to_copy) || !istype(spell_to_copy.holder))
 		return
+	if(is_type_in_list(spell_to_copy, forbidden_spells))
```

Comment:
I don't know exactly we started using this system, but typelists are a sin and this shouldn't have made it into new code.